src/EVM/Expr.hs

```diff
@@ -1616,3 +1614,25 @@ checkLHSConstProp a = isJust $ mapPropM_ lhsConstHelper a
 -- Commutative operators should have the constant on the LHS
 checkLHSConst :: Expr a -> Bool
 checkLHSConst a = isJust $ mapExprM_ lhsConstHelper a
+
+maybeLitByte :: Expr Byte -> Maybe Word8
```

Comment:
```suggestion
maybeLitByteSimp :: Expr Byte -> Maybe Word8
```

Reply:
Ayyyy you are so good. I'm sorry for wasting your time. Let me fix.